tests/common.py

```diff
@@ -303,26 +306,183 @@ def params_with_w_coords(w_coords):
 @dataclass
 class SplitForcefield:
     ref: Forcefield  # ref ff
-    scaled: Forcefield  # all charge terms scaled by 10x
-    inter_scaled: Forcefield  # intermolecular charge terms scaled by 10x
+    intra: Forcefield  # intermolecular charge terms scaled by 10x
+    solv: Forcefield  # water-ligand charge terms scaled by 10x
+    prot: Forcefield  # protein-ligand charge terms scaled by 10x
+    scaled: Forcefield  # all terms scaled by 10x
 
 
 def load_split_forcefields() -> SplitForcefield:
     """
     Returns:
-        OpenFF 2.0.0 ff,
-        OpenFF 2.0.0 ff with all charge terms scaled by 10x,
-        OpenFF 2.0.0 ff with only intermolecular charge terms scaled by 10x.
+        SplitForcefield which contains the ff with various
+        terms scaled by a factor of 10.
     """
     ff_ref = Forcefield.load_from_file("smirnoff_2_0_0_ccc.py")
```

Comment:
nit: use DEFAULT_FF here or `Forcefield.load_default()` unless it is specific to this forcefield